src/events/2018/2018-March-Iceland-Reykjavik-JSConf Iceland.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "JSConf Iceland 2018",
+  "topic": "Anything remotely related to JavaScript and the community around it",
+  "url": "https://cfp.jsconf.is",
+  "twitter": "jsconfis",
+  "city": "Reykjavik",
+  "stateProvince": "",
+  "country": "Iceland",
+  "eventStartDate": "2018-03-01T00:00:00.000Z",
+  "eventEndDate": "2018-03-02T00:00:00.000Z",
+  "cfpStartDate": "2018-10-05T00:00:00.000Z",
```

Comment:
CFP is 2017 right? I can update this if you'd like (but want to make sure I'm not making an incorrect assumption 😀)

Reply:
Whooooops yes!